### PR TITLE
adds setting to toggle announcement in bani overlay

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Screenshot 2019-08-29 at 7 39 24 PM

### CODE REVIEW

**Comment by maneetpaul** [comment]
Sorry to nitpick - can we capitalize the word "Gurmukhi" that is below "Bani Overlay"

File: app.js [hunk 1]
```diff
@@ -465,7 +465,11 @@ ipcMain.on('show-text', (event, arg) => {
       Transliteration: '',
     },
   };
-  showLine(textLine);
+
+  const announcementOverlay = store.getUserPref('app.announcement-overlay');
```
Comment (tsingh777) [inline app.js]: need an entry in `defaults.json` ?

Comment (inderpreetsingh) [reply app.js]: It's already there.

File: www/js/insert-slide.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ const electron = require('electron');
 const analytics = electron.remote.getGlobal('analytics');
 const tingle = require('./vendor/tingle');
 const strings = require('./strings');
+const settings = require('./settings');
+
+const { store } = electron.remote.require('./app');
+
+let announcementOverlay = store.getUserPref('app.announcement-overlay');
```
Comment (tsingh777) [inline www/js/insert-slide.js]: something buggy with this. It only seems to retrieve the defaults value. The checkbox is checked everytime the modal is opened regardless of the value in the setting. ideally they should be in sync.

File: www/js/insert-slide.js [hunk 3]
```diff
@@ -52,6 +57,17 @@ const langSlider = '<div class="lang-switch">'.concat(
   '</div>',
   '</td>',
   '</tr>',
+  '<tr>',
+  '<td>',
+  '<span class="lang-text">Show Announcement in Overlay</span>',
```
Comment (tsingh777) [inline www/js/insert-slide.js]: the 2 switches could use some `padding-right` imo. 
image

**Pull request merged**